deployment/docker-compose.latest.yml

```diff
@@ -84,6 +84,10 @@ services:
   msrc-goerli:
     <<: *defaults
 
+  registration-goerli:
```

Comment:
Do we also need `registration-goerli-backup`?

Reply:
No. That one doesn't register any interesting URL because it's only used for the MS, which doesn't need the URL.